docs/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,129 @@
+[Yubikey](https://www.yubico.com/) due to its reputation for strong security characteristics and multitude of form
+factors. Refer to
+[the documentation for more information on using a hardware device with the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root).
+
+### 2. Answer security questions and complete contact details
+
+When setting up a new account, AWS asks for contact information and security questions. Unfortunately, there is no API or automation available for this functionality. In the AWS console, visit the [Account settings](https://console.aws.amazon.com/billing/home?#/account) page and complete the _Alternate Contacts_ and _Configure Security Challenge Questions_ questions.
+
+
+### 3. Delete default VPCs and rules from default security groups
+
+Use [cloud-nuke](https://github.com/gruntwork-io/cloud-nuke) to remove the rules from the default VPC and the default ingress/egress rules from the default security groups. Note that it isn’t possible to actually delete the default security group, so instead the command deletes the rules, eliminating the risk of something being mistakenly exposed.
+
+For each account (shared, security, dev, prod etc), run the following command:
+
+```
+cloud-nuke defaults-aws
+```
+
+### 4. Subscribe to `BenchmarkAlarmTopic` Alarms
+
+The Config alerts and CloudWatch Metric Alarms all go to the SNS topic `BenchmarkAlarmTopic`. Unfortunately, there is no way to automate
+subscribing to the SNS topic as each of the steps require validating the delivery target. For each deployed account, follow the steps outlined in
+the [AWS docs](https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html) to be notified by Email, Phone,
+or SMS for each of the alerts.
+
+You can also configure an automated system integration if you have a third party alerting system or central dashboard.
+Follow the steps in the [AWS
+docs](https://docs.aws.amazon.com/sns/latest/dg/sns-http-https-endpoint-as-subscriber.html) on how to add an HTTPS endpoint as a subscriber to the alerts.
+
+### 5. Manually maintain buckets to analyze in the `buckets_to_analyze` variable
+
+To set up Macie to analyze the desired S3 buckets, you’ll need to create a **Macie classification job**. Typically,
+you’ll want it to analyze all the buckets in the region. However, the terraform AWS provider does not support specifying
+all the buckets in a region - it requires that an explicit list of buckets be provided. Therefore, you’ll
+need to maintain an explicit list of buckets per region, namely in the variable `buckets_to_analyze`. Please read the
+[documentation](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/security/macie/variables.tf#L21-L30)
+for this variable in order to understand how to structure the list of buckets per region. Once the underlying issue in the
```

Comment:
What is the underlying issue in the Terraform AWS provider? (We could link to it for folks who are curious).

Reply:
I removed because was fixed 😓 the issue in GW's repo links to the issue in aws provider.

Reply:
I'd certainly vote to remove any extra text if we can. If the provider issue is no longer blocking us, then we don't need to mention it. 

If we still have a blocking issue on our side, we may need to mention that to users.